src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -0,0 +1,34 @@
+@echo off
+setlocal
+
+set PROJECT_DIR=%~1
+set DOTNET_CMD=%~2
+set TOOLRUNTIME_DIR=%~3
+set TOOLS_DIR=%~dp0
```

Comment:
To help eliminate confusion (which I had until I read the rest of this script), can you name this variable something else. Perhaps BuildToolsPackageDir.